src/Engine/Audio/SteamAudio.cpp

```diff
@@ -1,15 +1,16 @@
 #include "SteamAudio.hpp"
 
 SteamAudio::SteamAudio() {
-
+    context = nullptr;
+    environmentalRenderer = nullptr;
 }
 
 SteamAudio::SteamAudio(IPLContext * context, IPLhandle * environment) {
 
 }
 
 SteamAudio::~SteamAudio() {
-
+    delete context;
```

Comment:
Is context guaranteed not to be nullptr? (Since it's set to nullptr in constructor.)